### DISCUSSION TITLE
Package name `siesta`, possibility of transfer request

### DISCUSSION THREAD

**Discussion by canonic-epicure** [npm/feedback#540] [Registry] [answered]
Hello,

I've filed a support ticket with a request of the package name transfer possibility at https://www.npmjs.com/support. I received a confirmation number 1280683 but no replies. Because of that, I'm repeating the request here, perhaps it will draw more attention.

We've just released an 6.0.0-alpha version of our open-source testing tool, called Siesta. Siesta is under development since 2009 (roughly since the same time as `npm` itself). Versions 1-5 were proprietary, version 6 is MIT licensed. We are committed to make Siesta a great testing tool. It is currently published on npm as `@bryntum/siesta`.

https://siesta.works
https://github.com/bryntum/siesta
https://www.bryntum.com/blog/announcing-technical-preview-of-siesta-6/
https://bryntum.com/products/siesta

There is an already published npm package `siesta`. Last publish date is 8 years ago. It also does not have any dependent modules. There are very few downloads, I guess those are from the automation bots or something like that.

I wonder if it would be possible to transfer the `siesta` package name to us. I tried to contact the package's author via the github issue: https://github.com/goatslacker/siesta/issues/3 but did not receive any replies.

Can you may be try to contact the author via email? Or may be provide his/her email to us so we could do that? If the author will not be reachable, would it be possible to transfer the name?

Please advise.

Regards, Nickolay

P. S. You can contact me at: nickolay@bryntum.com

**Comment by MylesBorins** [comment, marked as answer]
hey @canonic-epicure 

Thanks for opening the issue. We get a very very large amount of transfer requests and only have so many support agents to review them. I apologize that this hasn't been responded to yet but I can assure you that the team will get to you when they can.

I'll talk with the team who is responsible for managing this and see if I can get an idea of what the backlog is looking like and see if it makes sense to update the disputes policy with details on how long one might expect a dispute to take to be resolved.

**Reply by canonic-epicure** [reply]
Thank you for reply! Its definitely not urgent, but quite important for us. Yes, will be great to know that the request is "in the pipeline" and will be processed eventually.

**Comment by ljharb** [comment]
I've pinged the author personally (via direct means), so they'll likely respond on that issue soon.

**Reply by canonic-epicure** [reply]
Thank you!

**Comment by arpitBhalla** [comment]
We were trying to publish a scoped package for [React Native Elements](https://github.com/react-native-elements/react-native-elements).  But there's already an organization name existed and there's no published package in it, We opened a ticket on npm Support, it’s more than 3 months now but they haven’t responded, thus we moved to GitHub package registry, but installation from GitHub registry requires authentication using PAT, which makes somewhat installation process complex. I was wondering if there is possibility of transfer org name to us? as we need to release v4 soon.

**Reply by arpitBhalla** [reply]
cc @MylesBorins @ljharb

**Reply by MylesBorins** [reply]
hey @arpitBhalla. We have a massive backlog of requests. As each one requires research and extensive work to determine if we can move forward with the request it is currently taking as long as 3 - 4 months to get to requests at this time.

I recognize this is extremely frustrating, and we are actively working on figuring out how to both improve the process and get more staff involved so we can move faster. With that said we can only move so fast, making a mistake on these transfers can have drastic ecosystem impact even if it is non-obvious.

**Reply by arpitBhalla** [reply]
Hey @MylesBorins if possible can you prioritise our request? It's been more than 4 month now, we urgently need to release next version.

**Reply by MylesBorins** [reply]
Hey @arpitBhalla, we are still working through the backlog. I've surfaced your request to figure out how far back it is in our backlog, but we do handle things in a first-come-first-serve manner. Hopefully we can get to it soon
